libraries/SITL/SITL.cpp

```diff
@@ -503,27 +562,88 @@ const AP_Param::GroupInfo SIM::var_gps[] = {
     // @Param: GPS2_BYTELOS
     // @DisplayName: GPS 2 Byteloss
     // @Description: Percent of bytes lost from GPS 2
+    // @Units: %
     // @User: Advanced
     AP_GROUPINFO("GPS2_BYTELOS",  33, SIM,  gps_byteloss[1],  0),
     // @Param: GPS2_NUMSATS
     // @DisplayName: GPS 2 Num Satellites
     // @Description: Number of satellites GPS 2 has in view
     AP_GROUPINFO("GPS2_NUMSATS",  34, SIM,  gps_numsats[1],   10),
+    // @Param: GPS2_GLITCH
+    // @DisplayName: GPS 2 Glitch
+    // @Description: Glitch offsets of simulated GPS 2 sensor
+    // @Vector3Parameter: 1
+    // @User: Advanced
     AP_GROUPINFO("GPS2_GLTCH",    35, SIM,  gps_glitch[1],  0),
+    // @Param: GPS2_HZ
+    // @DisplayName: GPS 2 Hz
+    // @Description: GPS 2 Update rate
+    // @Units: Hz
     AP_GROUPINFO("GPS2_HZ",       36, SIM,  gps_hertz[1],  5),
+    // @Param: GPS2_DRFTALT
+    // @DisplayName: GPS 2 Altitude Drift
+    // @Description: GPS 2 altitude drift error
+    // @Units: m
+    // @User: Advanced
     AP_GROUPINFO("GPS2_DRFTALT",  37, SIM,  gps_drift_alt[1], 0),
+    // @Param: GPS2_POS
+    // @DisplayName: GPS 2 Position
+    // @Description: GPS 2 antenna phase center position relative to the body frame origin
+    // @Units: m
+    // @Vector3Parameter: 1
     AP_GROUPINFO("GPS2_POS",      38, SIM,  gps_pos_offset[1], 0),
+    // @Param: GPS2_NOISE
+    // @DisplayName: GPS 2 Noise
+    // @Description: Amplitude of the GPS2 altitude error
+    // @Units: m
+    // @User: Advanced
     AP_GROUPINFO("GPS2_NOISE",    39, SIM,  gps_noise[1], 0),
+    // @Param: GPS2_LCKTIME
+    // @DisplayName: GPS 2 Lock Time
+    // @Description: Delay in seconds before GPS2 acquires lock
+    // @Units: s
+    // @User: Advanced
     AP_GROUPINFO("GPS2_LCKTIME",  40, SIM,  gps_lock_time[1], 0),
+    // @Param: GPS2_ALT_OFS
+    // @DisplayName: GPS 2 Altitude Offset
+    // @Description: GPS 2 Altitude Error
+    // @Units: m
     AP_GROUPINFO("GPS2_ALT_OFS",  41, SIM,  gps_alt_offset[1], 0),
+    // @Param: GPS2_HDG
+    // @DisplayName: GPS 2 Heading
+    // @Description: Enable GPS2 output of NMEA heading HDT sentence or UBLOX_RELPOSNED
+    // @Values: 0:Disabled, 1:Enabled
+    // @User: Advanced
     AP_GROUPINFO("GPS2_HDG",      42, SIM,  gps_hdg_enabled[1], SIM::GPS_HEADING_NONE),
+    // @Param: GPS2_ACC
+    // @DisplayName: GPS 2 Accuracy
+    // @Description: GPS 2 Accuracy
+    // @Vector3Parameter: 1
+    // @User: Advanced
     AP_GROUPINFO("GPS2_ACC",      43, SIM,  gps_accuracy[1], 0.3),
+    // @Param: GPS_VERR
```

Comment:
Ah, this should be GPS2_VERR

Reply:
Ahh another mistake from me, 

This has also been fixed.